strawberryfields/apps/graph/dense.py

```diff
@@ -0,0 +1,179 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+r"""
+Dense subgraph identification
+=============================
+
+**Module name:** :mod:`strawberryfields.future.apps.graph.dense`
+
+.. currentmodule:: strawberryfields.future.apps.graph.dense
+
+The frontend module for users to find dense subgraphs. The :func:`find_dense` function
```

Comment:
Probably best to remove 'frontend', as it can now be confused with the SF frontend
```suggestion
Functions for finding dense subgraphs. The :func:`find_dense` function
```